workloads/virtual-machines/virtualized-hardware-configuration.md

```diff
@@ -90,6 +92,38 @@ spec:
         claimName: myclaim
 ```
 
+### Model
+
+Setting the CPU model is possible via `spec.domain.cpu.model`. The following VM will have a CPU with the `Conroe` model:
+
+```yaml
```

Comment:
Please make this example much smaller
please use ellipsis (…) to remove irrelevant parts

Reply:
I hope I understood you correct :smile: 
done